source/slang/slang-capabilities.capdef

```diff
@@ -1812,43 +1812,43 @@ alias GLSL_430_SPIRV_1_0 = _GLSL_430 + GLSL_420_SPIRV_1_0 | GLSL_420_SPIRV_1_0;
 //
 /// cuda 1.0 and related capabilities of other targets.
 /// [Version]
-alias cuda_sm_1_0 = _cuda_sm_1_0 | sm_4_0;
+alias cuda_sm_1_0 = _cuda_sm_1_0;
```

Comment:
Is there a reason why the `cuda_sm_1_0` does not need to include any `sm_*_*`, while others do have at least one (either directly or inherited)?

Reply:
It isn't bothering to loop in anything else because even the lowest supported HLSL `sm_*_*` value (which is `sm_4_0`) would wrap around and require a higher CUDA SM version than `_cuda_sm_1_0`.

Putting this PR together has made me realize that there are some really questionable things going on around how the Slang compiler/codebase is handling capabilities/profiles/etc., and a more thorough scrubbing is really called for. I just don't want to have the issue that this PR was originally trying to fix get stalled on a major overhaul like that.